base/pair.jl

```diff
@@ -1,5 +1,32 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Pair(x, y)
+    x => y
+
+Construct a `Pair` object with type `Pair{typeof(x), typeof(y)}`. The elements
+are stored in the fields `first` and `second`. They can also be accessed via
+iteration.
+
+See also [`Dict`].
```

Comment:
Since `Dict` has a docstring you can link to it with `@ref` as follows: ``[`Dict`](@ref)``